# Goal
# (Nenhum objetivo espec√≠fico para pouso)

---
# Result
bool success

---
# Feedback
float32 current_altitude
